### DISCUSSION TITLE
How is it that Libreddit is still working?

### DISCUSSION THREAD

**Discussion by ghost** [spikecodes/libreddit#827] [Q&A]
I used Libreddit today and it still works, despite the API being cut off.  How is it working?
a)Is it still using the paid API, meaning I should avoid using it too much so that whoever's running the instance doesn't get charged a lot?

b)Has it already been rewritten to use a webscraper or the private API?

c)Is it something else?

**Comment by sigaloid** [comment]
Here's a comment with more info: https://github.com/libreddit/libreddit/issues/785#issuecomment-1616138008
